FEXCore/Source/Interface/Core/OpcodeDispatcher.cpp

```diff
@@ -35,6 +35,7 @@ using X86Tables::OpToIndex;
 
 #define OpcodeArgs [[maybe_unused]] FEXCore::X86Tables::DecodedOp Op
 
+template<bool IsInt80>
```

Comment:
Probably worth choosing a different name for this, as it will be true even on windows, where the syscall immediate is 0x2E